src/custom/components/Page/index.tsx

```diff
@@ -58,9 +59,31 @@ export const Content = styled.div`
     margin: 24px auto;
   }
 
+  #table-container {
```

Comment:
Where is this used?

Reply:
In the cookie policy, I moved it out of the page, since it's not general enough https://github.com/gnosis/gp-swap-ui/pull/470/commits/7312ee53860fa3acf9e956ad52d58ff3994111b5